stack.yaml

```diff
@@ -3,6 +3,14 @@
 # * https://github.com/commercialhaskell/stackage/issues/4731
 resolver: nightly-2019-08-07
 packages: [.]
-extra-deps: [ghc-lib-parser-8.8.1.20191204]
+extra-deps: [ghc-lib-parser-8.8.2, ghc-lib-parser-ex-8.8.2]
```

Comment:
Are these both in the latest stackage nightly? If so, can they be deleted from here - and potentially the whole stack.yaml deleted?

Reply:
That's a good question. `https://www.stackage.org/nightly-2020-01-21` :
![Screen Shot 2020-01-21 at 6 08 37 AM](https://user-images.githubusercontent.com/1500167/72800075-81a04c00-3c14-11ea-8857-22a35e742568.png)
I'll  delete the extra deps. It would be helpful for me to retain the stack.yaml though for my hlint development if that's ok?

Reply:
My standard approach is to generate a stack with `stack init`, and put that on gitignore, so then people who want to dev with stack can, and my CI can always test it works with stackage head. Does that work for you?

Reply:
Yes, that would be fine. I'll delete stack.yaml accordingly.